docs/overview.md

```diff
@@ -1,6 +1,35 @@
-# Overview
+page_title: Registry Service 2.0
+page_description: Explains how to deploy a registry service
+page_keywords: registry, service, images, repository
 
-**TODO(stevvooe):** Table of contents.
+# Registry Service 2.0
 
-**TODO(stevvooe):** Include a full overview of each component and dispatch the
-user to the correct documentation.
+**TODO(stevvooe):** Include a full overview of each component and dispatch the user to the correct documentation.
+
+
+
+### Upgrading
+
+The Docker Registry Service 2.0 is backward compatible with images created by the 1.0 specification. If you are migrating a private registry to version 2.0, you should use the following process:
```

Comment:
Indeed, this is inaccurate. 
Compat will involve #329 and similar helpers, and more documentation work that unfortunately I'll have to find time for.
Let's leave this section as TODO for now.